verilog/formatting/tree_unwrapper.cc

```diff
@@ -3123,6 +3131,36 @@ void TreeUnwrapper::ReshapeTokenPartitions(
       break;
     }
 
+    case NodeEnum::kRegisterVariable:
+    case NodeEnum::kInstantiationType: {
+      size_t partition_size = partition.Children().size();
+      if (partition_size >= 4) {
```

Comment:
```suggestion
      if (partition_size < 4) break;
```
Early returns (or breaks) help the reader focus on the important part of the code. They also help avoid rightward drift.
More importantly, why is this `4` here? Could use an explanation. I know that you later access children `0..3`, but what does this mean in relation to actual Verilog code?